addons/project/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,162 @@
+---Fields in module 'project'---
+project      / project.project          / date (date)                   : NEW
+project      / project.project          / date_start (date)             : NEW
+project      / project.project          / label_tasks (char)            : NEW
+# New fields ---> Nothing to do
+
+project      / project.project          / members (many2many)           : DEL relation: res.users
+# Field removed ---> Nothing to do
```

Comment:
Project team is a relevant information; OCA should provide a module to keep this info.